spec/index.html

```diff
@@ -1564,12 +1604,34 @@ <h4>Delete Insert Operation</h4>
           <h4>Load Operation</h4>
           <div class="defn">
             <p><strong><span class="doc-ref" id="defn_loadOperation">Definition: Load Operation</span></strong></p>
-            <p>A <a href="#load">Load Operation</a> OpLoad is an Update Operation in which new triples (from a remote graph) are added in the Graph Store, either in the default slot or in a named
-            slot, if specified.</p>
-            <p>OpLoad ( GS, <em>documentIRI</em>) = Dataset-UNION ( GS, { graph ( <em>documentIRI</em> ) } )</p>
-            <p>OpLoad ( GS, <em>documentIRI</em>, <em>iri</em> ) = Dataset-UNION(GS, { {}, ( <em>iri</em>, graph ( <em>documentIRI</em> ) ) } )</p>
-            <p>where graph(<em>documentIRI</em>) is a function returning the RDF graph serialized by the RDF document retrieved from IRI <em>documentIRI</em>, where blank nodes present in the
-            retrieved graph are supposed to be "standardized apart"; i.e., blank nodes from a loaded graph need to be disjoint with the blank nodes already present in the Graph Store GS.</p>
+            <p>
+              A <a href="#load">Load Operation</a> `OpLoad` is an Update Operation in which new
+              triples (from a remote graph or dataset) are added to the Graph Store, either in the
+              default slot or in a named slot, if specified.
```

Comment:
What is a "slot"? These seem to be stand-ins for "(named) graph". If "(named) graph" can't be used yet, then this should be changed to --
```suggestion
              triples (from a remote graph or dataset) are added to the Graph Store.
```

Reply:
If "(named) graph" _can_ be used, then this should be changed to -- 
```suggestion
              triples (from a remote graph or dataset) are added to the Graph Store, either in the
              default graph or in a named graph, if specified.
```

Reply:
"slots" are described in [section 2](https://www.w3.org/TR/sparql12-update/#graphStore) and [4.2 Auxiliary Definitions](https://www.w3.org/TR/sparql12-update/#defn_graphStore).

The text "either in the default slot or in a named slot, if specified." in LOAD is unchanged from SPARQL 1.1 Update.

A Graph Store is a mutable container of slots (one unnamed slot, and zero or more slots with name). Each slot holds an immutable graph.

A dataset is strictly immutable, just like graphs aren't mutable.

The spec is a bit weak : https://www.w3.org/TR/sparql12-update/#defn_updateOperation says

> transforms a Graph Store GS to another Graph Store GS'
written
> Op(GS, Args) = GS'

but it is the same Graph Store with different contents.

See the [note](https://www.w3.org/TR/sparql12-update/#h-note-1) in [4.2 Auxiliary Definitions](https://www.w3.org/TR/sparql12-update/#formalModelAuxiliary) that mentions to punning on "Graph Store" and "RDF Dataset".

Not this PR!

Reply:
My bad. I should have looked to the pr-preview where it's much clearer why "slots" are being discussed.